utils/node_local_test.py

```diff
@@ -1272,6 +1360,19 @@ def main():
     if server.stop() != 0:
         fatal_errors.fail()
 
+    # If running all tests then restart the server under valgrind.
+    # This is really, really slow so just do list-containers, then
+    # exit again.
+    if len(sys.argv) == 2 and sys.argv[1] == 'all':
+        server = DaosServer(conf, valgrind=True)
+        server.start()
+        pools = get_pool_list()
+        for pool in pools:
+            cmd = ['pool', 'list-containers', '--svc', '0', '--pool', pool]
+            run_daos_cmd(conf, cmd, valgrind=False)
+        if server.stop() != 0:
+            fatal_errors = True
```

Comment:
(pylint-redefined-variable-type) Redefinition of fatal_errors type from node_local_test.BoolRatchet to bool